specs/windows/windows_security_center.table

```diff
@@ -4,7 +4,7 @@ schema([
     Column("firewall", TEXT, "The health of the monitored Firewall (see windows_security_products)"),
     Column("autoupdate", TEXT, "The health of the Windows Autoupdate feature"),
     Column("antivirus", TEXT, "The health of the monitored Antivirus solution (see windows_security_products)"),
-    Column("antispyware", TEXT, "The health of the monitored Antispyware solution (see windows_security_products)"),
+    Column("antispyware", TEXT, "Deprecated (always 'Good').", hidden=True),
```

Comment:
Does it make sense to _also_ remove the underlying API call? (Maybe not -- I'm sure it will remain supported for years)

Reply:
I was under the impression that this value was still useful on old Windows versions that we still support, so I imagined we wanted to keep the API call there. If this is not at all useful, then we should probably remove the call too

Reply:
I'm totally fine to delete the underlying API call, due to the fact that spyware engine doesn't get updates anymore from Microsoft. Properly the only way to get informative diagnostic from this column is from environments that doesn't connect to the internet and run Windows 10 before version 1607 (release date: 2/8/2016).

Reply:
If we think it's useful on older windows versions, then we may as well leave it. We can always remove it as those phase out.